### PR TITLE
Allergies: add testcase for checking "allergicTo" 

### PR DESCRIPTION
**Pull request by rachelhornung**
One of the solutions I mentored returned True for all allergies that had a  score lower than the total test score. This test should catch this.

### CODE REVIEW

File: exercises/allergies/canonical-data.json [hunk 1]
```diff
@@ -66,6 +66,31 @@
               "result": false
             }
           ]
+        },
+        {
+          "description": "allergic to strawberries but not peanuts",
```
Comment (rpottsoh) [inline exercises/allergies/canonical-data.json]: I am wondering if it wouldn't be enough to just modify `"allergic to eggs in addition to other stuff"`.  It seems to me that the new case is attempting to test for the same thing, and doing a better job of it apparently.  I am curious to read what others think.  Generally speaking I think this is a good improvement to the exercise. :+1:   Thanks!

**Comment by rachelhornung** [comment]
I agree that allergic to eggs in addition to other stuff with a test for peanuts: false would be sufficient.
I opted for an additional test case since here I want to make sure that the test identified not only multiple allergies, but that some allergies were not diagnosed. Which wouldn't be reflecte by "allergic to eggs in addition to other stuff". But technically, just modifying the existing test case would catch the same problem

**Comment by rpottsoh** [comment]
We'll give this a couple of days.  If no has a strong opinion one way or the other I'll merge this as is.  Stay tuned....

**Comment by rpottsoh** [comment]
@rachelhornung thanks for contributing to this exercise, to make it a little bit better! 👍   I am going to go ahead and merge this.

**Pull request merged**